SalesforceCommerceCloud/pwa-kit
# Back-port Shopper Experience Base Components into Retail Template

This PR does 2 things. Ports the shopper experience component from `commerce-sdk-react` into the retail template. 2. Enabled the Shopper Experience api in commerce-api client.

How to test: 

Run the dev server and goto the /page-viewer path. It should display a hierarchical view of the `homepage-example` page.

Before merging, remove the test page.
